OutSystems/outsystems-ui
# ROU-4348: Accessibility improvements identified on team audit for the Progress Bar and Circle

This PR is for implement the accessibility improvements identified on team audit for the Progress Bar and Circle.

### What was happening
- Screen reader was not reading the Progress Bar and Circle content

### What was done
- Replaced the aria-label attribute from Progress Bar and Circle with the aria-labelledby referencing the Content id

### Test Steps
1. Go to a page with a Progress Bar or Progress Circle
2. Enable the screen reader
3. Check that the screen reader reads the Progress Bar/Circle percentage, content and the role "progressbar".

### Screenshots
![proegressbar_sr](https://github.com/OutSystems/outsystems-ui/assets/108938618/4552defe-74e7-44b6-b91d-ae13d9bd78a6)
![progresscircle_sr](https://github.com/OutSystems/outsystems-ui/assets/108938618/f15a449d-2b77-45bf-ab67-a77dcbf452f9)


### Checklist
